experimental/dds/tree2/src/feature-libraries/typed-schema/flexList.ts

```diff
@@ -3,28 +3,61 @@
  * Licensed under the MIT License.
  */
 
+/** Used by `markEager`and `isMarkedEager` */
+const flexListEager = Symbol("FlexList Eager");
+
+/** Mark the given object as an "eager" item in a `FlexList`, regardless of its type. */
```

Comment:
Is this intended to be exported by the package? Or is it just an internal implementation detail? If the former, I think it would be worth giving more details here (or linking to somewhere that has those details).

Reply:
Definitely not public exported. The `FlexList` type does show up in the public API, but the types of things that users put into it are well-defined. They shouldn't be using FlexLists as a tool outside of our APIs so they don't need this level of control.